src/CMakeLists.txt

```diff
@@ -60,3 +60,4 @@ endif()
 
 # add MQT alias
 add_library(MQT::${PROJECT_NAME} ALIAS ${PROJECT_NAME})
+set(CMAKE_CXX_FLAGS " ${CMAKE_CXX_FLAGS_INIT} -std=c++11 -fPIC")
```

Comment:
These flags should not be necessary.
Just copy the respective sections from the qmap or qcec project.

And there is a newline missing at the end of this file